packages/cli/src/License.ts

```diff
@@ -84,6 +84,9 @@ export class License {
 		const collectUsageMetrics = isMainInstance
 			? async () => await this.usageMetricsService.collectUsageMetrics()
 			: async () => [];
+		const collectPassthroughData = isMainInstance
+			? async () => await this.usageMetricsService.getActiveWorkflowIds()
+			: async () => ({});
```

Comment:
```suggestion
		const collectPassthroughData = isMainInstance
			? async () => ({activeWorkflowIds: await this.usageMetricsService.getActiveWorkflowIds()})
			: async () => ({});
```

Just a suggestion, and then we change the `getActiveWorkflowIds` to be just an array, since we might want to add other metrics in the future, and the method `getActiveWorkflowIds` seems weird IMO since it's returning an object with a single attribute, I would not expect this from it's name.